FirebaseMessaging/Sources/Public/FIRMessaging.h

```diff
@@ -379,6 +379,23 @@ NS_SWIFT_NAME(Messaging)
  */
 @property(nonatomic, readonly, nullable) NSString *FCMToken NS_SWIFT_NAME(fcmToken);
 
+/**
+ *  Asynchronously getting the default token.
+ *
+ *  @param completion The completion handler to handle the token request.
+ */
+
+- (void)tokenWithCompletion:
```

Comment:
Did the new public APIs get reviewed?

Reply:
not yet, I realize this could be pushed to master early than API review. I've removed the new public API and use the existing token API for test app testing.